src/components/common/Chart/BarChart.tsx

```diff
@@ -102,7 +108,7 @@ function BarChartContent(props: BarChartContentProps): JSX.Element {
       if (ctx) {
         const colors = generateTerrawareRandomColors(theme, chartLabels?.length || 0);
         chartRef.current = await newChart(locale, ctx, {
-          type: 'bar',
+          type: type || 'bar',
```

Comment:
This is a BarChart component.. we should probably create a generic Chart component instead of supporting new types in the BarChart?
My suggestion:
1. Create a new Chart.tsx in this directory which is essentially a copy of this BarChart.tsx
2. In this BarChart.tsx simply call Chart.tsx with type='bar' so we don't have to update all the other chart calls
3. you can optionally also create a PieChart.tsx in this directory.. and set showLegend=true